src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -57,7 +57,7 @@ case class DigitalPackEmailFields(
       "MandateID" -> directDebitMandateId.getOrElse("")
     )
     case _: CreditCardReferenceTransaction => List("Default payment method" -> "Credit/Debit Card")
-    case _ => Seq("Default payment method" -> "PayPal")
+    case _: PayPalReferenceTransaction => Seq("Default payment method" -> "PayPal")
```

Comment:
payment method is already being sent for the DigitalPackEmailFields. The values follow a different format `"Credit/Debit Card"`.

Reply:
ok

I just look at the rest of the file and discovered it's full of commented out JSON and TODOs 😨 however I don't think it's been further invalidated by this change

Reply:
got it! thanks for the explanation @rupertbates